dev-packages/e2e-tests/test-applications/node-exports-test-app/scripts/consistentExports.ts

```diff
@@ -55,17 +53,12 @@ const DEPENDENTS: Dependent[] = [
     package: '@sentry/serverless',
     exports: Object.keys(SentryServerless),
     ignoreExports: [
-      // Deprecated, no need to add this now to serverless
-      'extractTraceparentData',
-      'getModuleFromFilename',
       // TODO: Should these be exported from serverless?
       'cron',
       'enableAnrDetection',
       'runWithAsyncContext',
       'hapiErrorPlugin',
```

Comment:
@reviewers wdyt do any of these make sense in serverless?

Reply:
`hapiErrorPlugin` and `cron` are the two that seem not needed, `runWithAsyncContext` should be included though.

Maybe also `enableAnrDetection`? But that is going away with the integration, so we have a clear replacement that users can use (also no longer in docs anywhere).

Reply:
done, added `runWithAsyncContext` and `enableAnrDetection` for completentess. thankd!